docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -117,9 +121,9 @@ Create a variable called `mlContext` and initialize it with a new instance of `M
 
 [!code-csharp[CreateMLContext](../../../samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#3 "Create the ML Context")]
 
-Next, to setup for data loading initialize the `_textLoader` global variable in order to reuse it.  Notice that we are using a `TextReader`. When you create a `TextLoader` using a `TextReader`, you pass in the context needed and the <xref:Microsoft.ML.Data.TextLoader.Arguments> class which enables customization. Specify the data schema by passing an array of <xref:Microsoft.ML.Data.TextLoader.Column> objects to the `TextReader` containing all the column names and their types. We defined the data schema previously when we created our `TaxiTrip` class.
+Next, to setup for data loading initialize the `_textLoader` global variable in order to reuse it. When you create a `TextLoader`, you pass in the context needed and the <xref:Microsoft.ML.Data.TextLoader.Arguments> class which enables customization. Specify the data schema by passing an array of <xref:Microsoft.ML.Data.TextLoader.Column> objects to the `TextReader` containing all the column names and their types. We defined the data schema previously when we created our `TaxiTrip` class.
 
-The `TextReader` class returns a fully initialized <xref:Microsoft.ML.Data.TextLoader>  
+The `TextLoader` class returns a fully initialized <xref:Microsoft.ML.Data.TextLoader>  
```

Comment:
Also xrefs `<xref:Microsoft.ML.Data.IDataView>` should be updated to a new namespace.

Reply:
Thanks for the feedback. Forgot to add WIP tag to it. I'll make the appropriate changes.